glitter/pages/middleware.py

```diff
@@ -3,6 +3,9 @@
 from django.http import Http404, HttpResponseRedirect
 
 from glitter.exceptions import GlitterRedirectException, GlitterUnpublishedException
+from django.conf import settings
+from glitter import urls as glitter_urls
+import importlib
```

Comment:
No, I think `importlib` is standard python library and should be above the `Django` lib